delivery_cttexpress/views/delivery_cttexpress_view.xml

```diff
@@ -72,6 +72,10 @@
                                 name="cttexpress_document_offset"
                                 attrs="{'required': [('delivery_type', '=', 'cttexpress')]}"
                             />
+                            <field 
+                                name="cttexpress_label_delay"
+                                attrs="{'required': [('delivery_type', '=', 'cttexpress')]}"
```

Comment:
I think the required thing doesn't work with integers.

Reply:
And what about this? In fact, you can put a 0 anyway here.

Reply:
I see. It looks like the UI takes care of it already avoiding falsey values